pages/common/dcfldd.md

```diff
@@ -0,0 +1,12 @@
+# dcfldd
+
+> Enhanced version of dd for forensics and security.
+> More information: <http://dcfldd.sourceforge.net/>.
+
+- Copy a disk to a raw image file and hash the image using SHA256:
+
+`dcfldd if=/dev/{{disk_device}} of={{file.img}} hash=sha256 hashlog={{file.hash}}`
+
+- Copy a disk to a raw image file, hashing each 1GB chunk:
+
+`dcfldd if=/dev/{{disk_device}} of={{file.img}} hash={{hash_algorithm}} hashlog={{file.hash}} hashwindow={{1G}}`
```

Comment:
What are the hash algorithms available ? Can we list the options directly here ?

Reply:
How do you suggest we list them ?

Reply:
What about this? `{{sha512|sha384|sha256|sha1|hd5}}`

Reply:
I guess it's a good idea yeah. Do you think I should submit another pull request @sbrl  ?

Reply:
@sidahmed-malaoui, I am not @sbrl but probably Sbrl will be agree with me: Yes! Of course, feel free to open a PR for this change. Thank you in advance!

Reply:
Yeah, another PR would be great @sidahmed-malaoui :-)

Reply:
Okay guys. Pull request coming.